indexing-hadoop/src/main/java/org/apache/druid/indexer/Utils.java

```diff
@@ -143,4 +151,66 @@ public static String getFailureMessage(Job failedJob, ObjectMapper jsonMapper)
       return null;
     }
   }
+
+  /**
+   * When the JobHistory server is unavailable, it's possible to fetch the application's status
+   * from the YARN ResourceManager instead.
+   */
+  public static boolean checkAppSuccessFromYarnRM(Job job) throws Exception
+  {
+    log.info("Could not retrieve job status from JobHistory server, trying YARN ResourceManager instead.");
```

Comment:
nit: this probably belongs to whoever called this method .. can't make assumption of JobHistory failure here.

Reply:
I had originally intended for that to only be called on Job IOExceptions, but moved this logging into `checkAppSuccessForJobIOException` with some rewording